core/src/main/java/cz/xtf/core/bm/BuildManager.java

```diff
@@ -16,6 +17,10 @@ public BuildManager(OpenShift openShift) {
         if (openShift.getProject(openShift.getNamespace()) == null) {
             openShift.createProjectRequest();
             openShift.waiters().isProjectReady().waitFor();
+            // Adding a label can be only done via 'namespace'. It cannot be set via 'project' API.
+            // Thus we do this separately. Also, updating namespaces seems to be restricted only to Admin users.
+            OpenShifts.admin().namespaces().withName(openShift.getNamespace()).edit().editMetadata().addToLabels(
+                    OpenShift.XTF_GENERATED_LABEL, "true").endMetadata().done();
```

Comment:
It might be useful to have label either for xtf in generall (`xtf.cz/generatedNamespace`) but also for different type of namespaces - `xtf.cz/buildNamespace`, `xtf.cz/testNamespace` so that we can differ them easily.

Usecase: I want to easily delete all test namespace but no build namespace, since I still want to share some resources. Not sure if this scenario is valid, though.

Just thinking in long term...WDYT?

Reply:
This looks like a good point. Although, I would probably keep it for the time we will really need it.

At the moment, I'll update this label to what Martin has proposed - `xtf.cz/managed`. :)